ui/raidboss/data/02-arr/alliance/the_world_of_darkness.ts

```diff
@@ -0,0 +1,111 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  mainTank?: string;
+  sullenDebuff?: boolean;
+  irefulDebuff?: boolean;
+}
+
+// TODO:
+//  Angra Mainyu
+//    Add Level 100 Flare
+//    Add Level 150 Doom
+//    Add Roulette?
+//    Add info text for add spawns?
+//  Five-Headed Dragon
+//  Howling Atomos
+//  Cerberus
+//  Cloud of Darkness
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheWorldOfDarkness,
+  triggers: [
+    {
+      id: 'Angra Mainyu Auto',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: 'DB4', source: 'Angra Mainyu' }),
+      run: (data, matches) => data.mainTank = matches.target,
+    },
+    {
+      id: 'Angra Mainyu Gain Sullen',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: '27c' }),
+      condition: Conditions.targetIsYou(),
+      run: (data) => data.sullenDebuff = true,
+    },
+    {
+      id: 'Angra Mainyu Lose Sullen',
+      type: 'LosesEffect',
+      netRegex: NetRegexes.losesEffect({ effectId: '27c' }),
+      condition: Conditions.targetIsYou(),
+      run: (data) => data.sullenDebuff = false,
+    },
+    {
+      id: 'Angra Mainyu Gain Ireful',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: '27d' }),
+      condition: Conditions.targetIsYou(),
+      run: (data) => data.irefulDebuff = true,
+    },
+    {
+      id: 'Angra Mainyu Lose Ireful',
+      type: 'LosesEffect',
+      netRegex: NetRegexes.losesEffect({ effectId: '27d' }),
+      condition: Conditions.targetIsYou(),
+      run: (data) => data.irefulDebuff = false,
+    },
+    {
+      id: 'Angra Mainyu Double Vision',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: 'CC8', source: 'Angra Mainyu', capture: false }),
+      alertText: (data, _matches, output) => {
+        if (data.sullenDebuff) {
+          // Stand behind boss in the red half to switch to Ireful
+          return output.red!();
+        } else if (data.irefulDebuff) {
+          // Stand in front of boss in the white half to switch to Sullen
+          return output.white!();
+        } else if (data.mainTank === data.me) {
+          // Main tank should stand in front
+          return output.white!();
+        }
+        // Everyone else generally stand behind
+        return output.red!();
```

Comment:
Hmm.  Sorry to quibble, but I think tracking the main tank is overthinking it.  The boss is frozen in place during this mechanic and I'm not sure that there's a reason for anybody to stand in any particular place on the first one.

What about "stand in either color" as a no debuff text? I think that's better than "double vision".

Reply:
Yea that is probably overkill. I'm also happy to just leave the message off. I guess its not that critical it was more of a thought that it might feel weird not to get a pop up at all. The more I think about it, the more keeping it simple seems fine. It would be weird to have to go "oh I guess I don't do anything special". So I'll just go ahead and remove this.